background_scripts/JobQueue.js

```diff
@@ -0,0 +1,118 @@
+class JobDeamon {
```

Comment:
```suggestion
class JobDemon {
```

(or `daemon`: https://www.differencebetween.com/difference-between-daemon-and-vs-demon/)

Reply:
:D